share/jupyterhub/templates/not_running.html

```diff
@@ -18,8 +18,10 @@ <h1>
       <p>
         {% if failed %}
         The latest attempt to start your server {{ server_name }} has failed.
-        {% if failed_message %}
-          {{ failed_message }}
+        {% if failed_html_message %}
+          <p>{{ failed_html_message | safe }}</p>
+        {% elif failed_message %}
+          <p>{{ failed_message }}</p>
```

Comment:
I got thinking a bit about having a p tag within a p tag, but seems okay. Seems a bit off having parts of this section in dedicated p tags while others aren't etc but I don't really know the consequences.

If this feels OK then this LGTM.

Reply:
Good catch, fixed by closing the previous p before opening the new one.